src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/FakeContentTypeRegistryService.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Formatting
+{
+    internal class FakeContentTypeRegistryService : IContentTypeRegistryService
+    {
+        private readonly object _lock = new object();
+        private readonly Dictionary<string, ImmutableArray<string>> _contentTypeDefinitions = new Dictionary<string, ImmutableArray<string>>();
+        private readonly Dictionary<string, IContentType> _contentTypes = new Dictionary<string, IContentType>();
+
+        public FakeContentTypeRegistryService()
+        {
+            // Standard content types normally defined in BufferFactoryService
```

Comment:
Given Razor is pretty well scoped it might make sense to just add Razor's expected content types to this list to avoid the test boiler plate

Reply:
If we keep these types, I would prefer to set them up outside the constructor except for items provided by the actual instance of the service.